spec/dynamic_time_zone/time_with_zone_patch_spec.rb

```diff
@@ -45,18 +61,18 @@
       DynamicTimeZone.enabled = false
     end
 
-    it "strftime does not raise error with timezone #{low_offset_timezone}" do
-      Time.zone = low_offset_timezone
+    it "strftime does not raise error with timezone #{low_positive_offset_timezone}" do
+      Time.zone = low_positive_offset_timezone
       expect { strftime_now }.not_to raise_error
     end
 
-    it "default to_s ends with +1000 with timezone #{low_offset_timezone}" do
-      Time.zone = low_offset_timezone
+    it "default to_s ends with +1000 with timezone #{low_positive_offset_timezone}" do
+      Time.zone = low_positive_offset_timezone
```

Comment:
Can you add more test to assert the 

`default to_s ends with -1000 with timezone #{low_negative_offset_timezone}`

Reply:
Done